.gitignore

```diff
@@ -22,6 +22,9 @@ jspm_packages
 # Serverless directories
 .serverless
 /oss-cloud-backend/.env
+*.pem
 
 #database
 .dynamodb
+oss-cloud-backend/config/local.yml
+oss-cloud-backend/config/dev.yml
```

Comment:
I'm not sure do we want to ignore these files?

Reply:
Deleted from gitignore.